Foundation/NSRunLoop.swift

```diff
@@ -8,8 +8,8 @@
 //
 
 
-public let NSDefaultRunLoopMode: String = "" // NSUnimplemented
-public let NSRunLoopCommonModes: String = "" // NSUnimplemented
+public let NSDefaultRunLoopMode: String = "NSDefaultRunLoopMode"
+public let NSRunLoopCommonModes: String = "NSRunLoopCommonModes"
```

Comment:
These probably will need to be the same as their CF counterparts when run loops will be supported.